actionmailer/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Rails 6.0.0.alpha (Unreleased) ##
 
+*   Add `Base.unregister_observer`, `Base.unregister_observers`,
+    `Base.unregister_interceptor`, `Base.unregister_interceptors`,
+    `Base.unregister_preview_interceptor` and `Base.unregister_preview_interceptors`.
+    This makes it possible to dynamically add and remove email observers and
+    interceptors at runtime in the same way they're registered.
+
+    *Kota Miyake*
```

Comment:
Since this builds on https://github.com/rails/rails/pull/18446, could you add the original author's name here too?